gcp-cs-gke/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Pulumi.Gcp.Container;
+using Pulumi.Gcp.Container.Inputs;
+using Pulumi.Gcp.Container.Outputs;
+
+class Program
+{
+    static Task<int> Main()
+    {
+        return Deployment.RunAsync(async () =>
+        {
+
+            var name = "helloworld";
+
+            var config = new Config();
+
+            var latestMasterVersion = await Pulumi.Gcp.Container.Invokes.GetEngineVersions(new GetEngineVersionsArgs { });
```

Comment:
Nit: You don't need to fully quality `Invokes` in this case (until you import another namespace)